### ISSUE TITLE
Freaking speed bumps

### ISSUE THREAD

**Issue by aubergine10** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#68]
Similar sort of thing as #67 (speed cams).

User plops a prop, we recognise it as a speed bump, and it affects reckless (and on-call emergency) drivers on that road, forcing to slow to the speed limit.

Again, just jotting down random idea.

**Comment by FireController1847** [comment]
If you're going 90 MPH and the speed limit is 80 MPH, I would love to see a car slow down from 90 to 80 to see them go over that speed bump, hahaha

Just poking fun at your wording. I like this idea. Maybe instead of plopping a prop we could add a junction function like crosswalks?

**Comment by aubergine10** [comment]
Yes, good point. Although how would user choose their preferred style of speed bump each time? In UK we have 4 or 5 common styles of speed bump, so there would need to be way to choose.

That's why I was thinking of maybe integrate with Find It, so that when user plops an asset on a road segment, we can identify it as speed bump and act accordingly. I think this would be very intuitive for users, rather than a dedicated tool.

That being said, if we implement #51, we could have a "Speed bumps" tab showing icons for all speed bumps, so it's a dedicated tool but made to look like a "pick and plop" find it tool. Best of both worlds!

**Comment by FireController1847** [comment]
I mean, the game has built in sliders right? (ex. Window Transparency option). We could use sliders to set what we want the maximum speed set to (in the U.S. it's most commonly 15 mph). Somewhat related, there's also something in the U.S. called "speed humps," which are much larger and wider than speed bumps.
